docs/design/guardrails/tasks/test_cases.md

```
@@ -0,0 +1,90 @@
+## Guardrails Integration Test Cases
+
+---
+test_suite: guardrails_test.go
```

markdownlint-cli2 (0.23.0)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)